cypress_shared/pages/manage/lostBedCreate.ts

```diff
@@ -14,8 +14,6 @@ export default class LostBedCreatePage extends Page {
   }
 
   public completeForm(lostBed: LostBed): void {
-    cy.get('input[name="lostBed[numberOfBeds]"]').type(lostBed.numberOfBeds.toString())
```

Comment:
Should we not use `ApprovedPremisesLostBed` which does have the number of beds on it? Or are we deprecating that?

Reply:
Oops, yes. That was a bit of a slapdash effort on my part. I think it will be deprecated eventually, but not now